utils/formats/simplejsonapi.go

```diff
@@ -22,6 +22,8 @@ type SimpleJsonResults struct {
 	SecretsViolations         []SourceCodeRow               `json:"secretsViolations"`
 	IacsViolations            []SourceCodeRow               `json:"iacViolations"`
 	SastViolations            []SourceCodeRow               `json:"sastViolations"`
+	MaliciousVulnerabilities  []SourceCodeRow               `json:"malicious_code"`
+	MaliciousViolations       []SourceCodeRow               `json:"maliciousViolations"`
```

Comment:
```suggestion
	MaliciousVulnerabilities  []SourceCodeRow               `json:"maliciousCode"`
	MaliciousViolations       []SourceCodeRow               `json:"maliciousViolations"`
```
make sure the formats are the same...

Reply:
ok